manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.locale.en-US.yaml

```diff
@@ -0,0 +1,42 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Streamlink.Streamlink
+PackageVersion: 7.0.0-1
+PackageLocale: en-US
+Publisher: Streamlink
+PublisherUrl: https://github.com/streamlink/streamlink
+PublisherSupportUrl: https://github.com/streamlink/streamlink/issues
+Author: Streamlink
+PackageName: Streamlink
+PackageUrl: https://streamlink.github.io/
+License: BSD-2-Clause
+LicenseUrl: https://github.com/streamlink/windows-builds/blob/HEAD/LICENSE
+Copyright: Copyright (c) 2016-2022, Streamlink Team
+CopyrightUrl: https://raw.githubusercontent.com/streamlink/streamlink/master/LICENSE

```

Comment:
Incorrect copyright date and license data. The license data of the Streamlink project should be used, not the one of the Windows builds repo, which is just a bunch of BASH scripts for building installers and portable archives.